incubator/binding-risingwave/src/main/java/io/aklivity/zilla/runtime/binding/risingwave/internal/statement/RisingwaveCommandGenerator.java

```diff
@@ -16,31 +16,29 @@
 
 import java.util.List;
 
-import net.sf.jsqlparser.statement.Statement;
 import net.sf.jsqlparser.statement.create.table.CreateTable;
 import net.sf.jsqlparser.statement.create.table.Index;
 
-public abstract class CommandGenerator
+public abstract class RisingwaveCommandGenerator
```

Comment:
I tend to think of each of these as a `template` instead of a `generator` but it's still fine to have `generate` method even if we rename them to `template`.